tests/staggered_gauge_utils.cpp

```diff
@@ -300,10 +300,10 @@ void computeStaggeredPlaquetteQDPOrder(void** qdp_link, double plaq[3], const Qu
   gauge_param.cpu_prec = gauge_param_in.cpu_prec;
 
   gauge_param.cuda_prec = gauge_param_in.cuda_prec;
-  gauge_param.reconstruct = QUDA_RECONSTRUCT_NO;
+  gauge_param.reconstruct = gauge_param_in.reconstruct; //QUDA_RECONSTRUCT_NO;
```

Comment:
why leave the old buggy routine in the comment as opposed to just deleting this?  Else this will be confusing to anyone looking at this in the future.

Reply:
Fixed, duly noted!